config/optional/config_ignore.settings.yml

```diff
@@ -2,3 +2,7 @@ ignored_config_entities:
   - 'language.entity.*'
   - 'media.type.image:uuid'
   - 'crop.type.focal_point:uuid'
+  - 'cohesion.settings'
+  - 'purge.plugins'
+  - 'purge.logger_channels'
```

Comment:
@chandan-singh7929 Should we also add a update hook to update config ignore configs for the user who are already using Acquia CMS (or Custom profile) ?
cc @kdruckman

Reply:
Done